.travis.yml

```diff
@@ -1,11 +1,8 @@
 language: node_js
 node_js:
```

Comment:
as the time of writing, [nodejs' release page](https://nodejs.org/en/about/releases/) indicates the oldest "Active LTS version" is v10, should we drop all the version earlier than v10 at this time, since nodejs has already dropped it?

Reply:
Why drop, because of  jest？

Reply:
https://nodejs.org/en/about/releases/

Reply:
keep them if jest supports.

so many developper just on node8

Reply:
sure.